src/crimson/net/SocketMessenger.cc

```diff
@@ -165,9 +165,30 @@ SocketMessenger::bind(const entity_addrvec_t& addrs)
                           [this, addrs] (auto& count) {
     return seastar::repeat_until_value([this, addrs, &count] {
       assert(count >= 0);
+
```

Comment:
nit: in crimson we rarely use extra empty lines. I think it's preferable to strip this...